docs/server/source/data-models/transaction-model.rst

```diff
@@ -48,6 +48,7 @@ Here's some explanation of the contents:
   to all (including nested) keys of the ``"data"`` JSON document:
   1) keys (i.e. key names, not values) must **not** begin with the ``$`` character, and
   2) keys must not contain ``.`` or the null character (Unicode code point 0000).
+  3) The key `"language"` (at any level in the hierarchy) is a special key and used for specifying text search language. Its value must be one of the allowed values; see the valid [Text Search Languages](https://docs.mongodb.com/manual/reference/text-search-languages/) in the MongoDB Docs. In BigchainDB, only the languages supported by *MongoDB community edition* are allowed.
```

Comment:
I just noticed an error from a previous PR, but we can fix it here: It says `keys of the "data" JSON document` but that's only for `asset` (i.e. `asset.data`). If you remove the `"data"` then I think the sentence becomes true.

Reply:
There's a second problem here. This is an RST file but the Markdown link markup is being used for the link to https://docs.mongodb.com/manual/reference/text-search-languages/  It should be changed to the RST markup for external links.